CHANGES.md

```diff
@@ -227,6 +223,12 @@ breaking changes, and mappings for the large list of deprecated functions.
 
 [Migration guide]: https://github.com/openssl/openssl/tree/master/doc/man7/migration_guide.pod
 
+### Changes between 3.0.6 and 3.0.7 [?? ??? ????]
```

Comment:
Any reason for `?? ??? ????` and not the usual `xx XXX xxxx`?

Reply:
No reason. Just forgot that we do xx XXX xxxx instead. :grin: